XPlat/Shopify/iOS/source/Shopify.iOS/ApiDefinition.cs

```diff
@@ -0,0 +1,1964 @@
+		[Export ("taxable")]
+		bool Taxable { get; }
+
+		// @property (copy, nonatomic) NSDictionary * properties;
+		[Export ("properties", ArgumentSemantic.Copy)]
+		NSDictionary Properties { get; set; }
+
+		// @property (readonly, copy, nonatomic) NSString * fulfillmentService;
+		[Export ("fulfillmentService")]
+		string FulfillmentService { get; }
+
+		// -(instancetype)initWithVariant:(BUYProductVariant *)variant;
+		[Export ("initWithVariant:")]
+		IntPtr Constructor (ProductVariant variant);
+	}
+
+	// @interface BUYCartLineItem : BUYLineItem
+	[BaseType (typeof(LineItem), Name = "BUYCartLineItem")]
+	interface CartLineItem
+	{
+		// @property (readonly, nonatomic, strong) BUYProductVariant * variant;
+		[Export ("variant", ArgumentSemantic.Strong)]
+		ProductVariant Variant { get; }
+	}
+
+	// @interface BUYTheme : NSObject
+	[BaseType (typeof(NSObject), Name = "BUYTheme")]
+	interface Theme
+	{
+		// extern const CGFloat kBuyPaddingSmall;
+		[Static]
+		[Field ("kBuyPaddingSmall", "__Internal")]
+		nfloat BuyPaddingSmall { get; }
+
+		// extern const CGFloat kBuyPaddingMedium;
+		[Static]
+		[Field ("kBuyPaddingMedium", "__Internal")]
+		nfloat BuyPaddingMedium { get; }
+
+		// extern const CGFloat kBuyPaddingLarge;
+		[Static]
+		[Field ("kBuyPaddingLarge", "__Internal")]
+		nfloat BuyPaddingLarge { get; }
+
+		// extern const CGFloat kBuyPaddingExtraLarge;
+		[Static]
+		[Field ("kBuyPaddingExtraLarge", "__Internal")]
+		nfloat BuyPaddingExtraLarge { get; }
+
+		// extern const CGFloat kBuyTopGradientViewHeight;
+		[Static]
+		[Field ("kBuyTopGradientViewHeight", "__Internal")]
+		nfloat BuyTopGradientViewHeight { get; }
+
+		// extern const CGFloat kBuyCheckoutButtonHeight;
+		[Static]
+		[Field ("kBuyCheckoutButtonHeight", "__Internal")]
+		nfloat BuyCheckoutButtonHeight { get; }
+
+		// extern const CGFloat kBuyPageControlHeight;
+		[Static]
+		[Field ("kBuyPageControlHeight", "__Internal")]
+		nfloat BuyPageControlHeight { get; }
+
+		// extern const CGFloat kBuyBottomGradientHeightWithPageControl;
+		[Static]
+		[Field ("kBuyBottomGradientHeightWithPageControl", "__Internal")]
+		nfloat BuyBottomGradientHeightWithPageControl { get; }
+
+		// extern const CGFloat kBuyBottomGradientHeightWithoutPageControl;
+		[Static]
+		[Field ("kBuyBottomGradientHeightWithoutPageControl", "__Internal")]
+		nfloat BuyBottomGradientHeightWithoutPageControl { get; }
+
+
+		// @property (nonatomic, strong) UIColor * tintColor;
+		[Export ("tintColor", ArgumentSemantic.Strong)]
+		UIColor TintColor { get; set; }
+
+		// @property (assign, nonatomic) BUYThemeStyle style;
+		[Export ("style", ArgumentSemantic.Assign)]
+		ThemeStyle Style { get; set; }
+
+		// @property (assign, nonatomic) BOOL showsProductImageBackground;
+		[Export ("showsProductImageBackground")]
+		bool ShowsProductImageBackground { get; set; }
+
+
+		// interface BUYTheme_Additions
+
+		[Static]
+		[Export ("topGradientViewTopColor")]
+		UIColor TopGradientViewTopColor ();
```

Comment:
As suggestion, I would change these methods to properties or, if you decide to leave them as methods, add a Get prefix.